src/components/request/confirmRequest.js

```diff
@@ -48,7 +48,7 @@ class ConfirmRequest extends React.Component {
             value={link}
             className={styles.copy}/>
           <a className={styles.emailLink}
-            href={`mailto:?subject=My Lisk ID&body=Hey there, this is my Lisk ID: ${address}. Please send me ${amount} LSK.`}>
+            href={`mailto:?subject=My Lisk ID&body=Hey there, here is a link you can use to send me LSK via their wallet: ${link}`}>
```

Comment:
`mailto:?subject=Request ${amount} LSK to ${address}&body=Hey there, here is a link you can use to send me ${amount} LSK via your wallet: ${link}`